### PR TITLE
Don't allow creation without google login; Hide admin toolbar if not logged in as admin

### PR DESCRIPTION
**Pull request by alexhoffman617**

### CODE REVIEW

**Comment by alexhoffman617** [comment]
@samblissard @bjsawyer @cameronivey review plz

File: src/app/components/create-retro-form/create-retro-form.component.ts [hunk 1]
```diff
@@ -14,13 +14,23 @@ export class CreateRetroFormComponent {
   retroName: string;
   numberOfVotes = 2;
   phases = [];
-  af;
```
Comment (bjsawyer) [inline src/app/components/create-retro-form/create-retro-form.component.ts]: Still need `af` here I think.

File: src/app/components/create-retro-form/create-retro-form.component.ts [hunk 2]
```diff
@@ -14,13 +14,23 @@ export class CreateRetroFormComponent {
...
+  user;
 
-  constructor(af: AngularFire, private router: Router, private dialogRef: MdDialogRef<CreateRetroFormComponent>) {
-    this.af = af;
```
Comment (bjsawyer) [inline src/app/components/create-retro-form/create-retro-form.component.ts]: Same here

**Comment by cameronivey** [comment]
lets merge this in, i need to update my branch a bit to work with this

**Pull request merged**